Sophos/sophos-analysis-threat-center/ingest/parser.yml

```diff
@@ -0,0 +1,58 @@
+name: sophos-analysis-threat-center
+pipeline:
+  - external:
+      name: json.parse-json
+      properties:
+        input_field: "{{original.message}}"
+        output_field: message
+    name: parse_json
+
+  - name: set_ecs_fields
+  - name: set_sophos_fields
+
+stages:
+  set_ecs_fields:
+    actions:
+      - set:
+          "@timestamp": "{{parse_json.message.calendar_time}}"
+
+          event.kind: "event"
+          event.ingested: "{{parse_json.message.ingestion_timestamp}}"
```

Comment:
The definition of `event.ingested` is: 
```
Timestamp when an event arrived in the central data store.

This is different from @timestamp, which is when the event originally occurred. It’s also different from event.created, which is meant to capture the first time an agent saw the event.

In normal conditions, assuming no tampering, the timestamps should chronologically look like this: @timestamp < event.created < event.ingested.
```

Defining `event.ingested` with a timestamp in the event won't respect the chronology between dates.
I suggest removing this field
```suggestion
```